build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -95,6 +95,13 @@ jobs:
       TargetFolder: '$(Build.ArtifactStagingDirectory)\AppXContents'
       flattenFolders: false
 
+  - powershell: |
+      $fullpackagePath = "${{ parameters.fullnupkgdir }}"
+      $targetsFilePath = "$(Build.SourcesDirectory)\build\NuSpecs"
+      Copy-Item -Path "$targetsFilePath\ProjectReunion-Nuget-Native.targets" -Destination "$fullpackagePath\build\native\Microsoft.ProjectReunion.Foundation.targets"
+      Copy-Item -Path "$targetsFilePath\ProjectReunion-Nuget-Common.targets" -Destination "$fullpackagePath\build\Microsoft.ProjectReunion.targets"
```

Comment:
There's a Microsoft.ProjectReunion.Foundation.targets file also, are you purposefully excluding that?  It's a "meta" targets file that rings in MRRCore's targets file.

Reply:
Yep I am purposefully excluding it. The original nuspec used the file like this <file target="build\native\$ID$.targets" src="ProjectReunion-Nuget-Native.targets"/> and so ProjectReunion-Nuget-Native.targets would become Microsoft.ProjectReunion.Foundation.targets.

---
In reply to: [520765659](https://github.com/microsoft/ProjectReunion/pull/284#discussion_r520765659) [](ancestors = 520765659)